lib/pyspark_startup.py

```diff
@@ -0,0 +1,46 @@
+import findspark
+findspark.init()
+
+from pyspark.sql import SparkSession, DataFrame
+from pyspark.sql.functions import countDistinct, first
+
+def init() -> SparkSession:
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
+    return spark
+
+def load(spark: SparkSession) -> DataFrame:
```

Comment:
Myślałem też czy nie byłoby dobrze zachować orginalne tabele (title_basics, title_episode itd). Czasami wygodnie jest podejrzeć i sprawdzić czy czegoś nie zgubiliśmy przy przetwarzaniu.

Reply:
zachowywać nie ma sensu, bo są zbyt duże, ale może być to wskazówka, że dobrze mieć by małe metody ładujące. Przyczym ja bym preferował jedną metodą i np. klasę enum do przechowania tabel jakie umiemy zaczytać tym sposobem.
```
class FilmTables(Enum):
   BASICS,
  ...
def read_table(spark: SparkSession, table: FilmTables) -> DataFrame:
...
```

Reply:
Dodałem jako TODO (wtedy to by mogło się przepleść z dodaniem odpowiednich logów = 2 zadania)